docs/index.rst

```diff
@@ -20,7 +20,7 @@ A detailed configuration file sample (with all usable parameters) can be found *
    amplitude
    custom_model
    decay_topology
-   resonacnes_params
```

Comment:
It is the filename of rst. 
`git mv resonacnes_params.rst resonances_params.rst`

Reply:
Ohh, sorry yes I have changed the filename